docs/tutorial_step4.md

```diff
@@ -148,7 +148,7 @@ So we will update our request on environment in `activity/activity.js` to test t
 		}
 	});
 
-Let's display the JavaScript console on your browser and test it. Launch the Pawn activity from the **Start new** menu or from List view of activities. Here is the result:
+Let's display the JavaScript console on your browser and test it. Launch the Pawn activity from the **Start new** menu(hover mouse over activity icon, list will appear, click on **Start new**) or from List view of activities. Here is the result:
```

Comment:
We don't need this precision. It's a standard behavior in Sugarizer.

Reply:
Okay, make sense.